### PR TITLE
[4.0] RavenDB-11305

### PR DESCRIPTION
**Pull request by gregolsky**

### CODE REVIEW

File: src/Raven.Studio/typescript/commands/version/getLatestVersionInfoCommand.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        super();
+    }
+ 
+    execute(): JQueryPromise<VersionInfoDto> {
+        const args = { refresh: this.refresh };
+        const url = endpoints.global.buildVersion.buildVersionUpdates 
+            + (this.refresh ? this.urlEncodeArgs(args) : '');
+        return this.post<VersionInfoDto>(url, null)
+            .fail(response => {
+                this.reportError(`Failed to get latest version info.'`,
```
Comment (ml054) [inline src/Raven.Studio/typescript/commands/version/getLatestVersionInfoCommand.ts]: remove ' before ` ?

File: src/Raven.Server/Web/System/BuildVersionHandler.cs [hunk 2]
```diff
@@ -49,5 +54,51 @@ private static byte[] GetVersionBuffer()
...
+            return SystemTime.UtcNow - lastRunAt.Value <= LatestVersionCheckThrottlePeriod;
+        }
+
+        private async Task WriteVersionUpdatesInfo()
+        {
+            var versionUpdatesInfo = LatestVersionCheck.GetLastRetrievedVersionUpdatesInfo();
+            using (var context = JsonOperationContext.ShortTermSingleUse())
+            {
+                var stream = new MemoryStream();
+                using (var writer = new BlittableJsonTextWriter(context, stream))
```
Comment (ppekrol) [inline src/Raven.Server/Web/System/BuildVersionHandler.cs]: swap 'stream' with 'ResponseBodyStream()'

File: src/Raven.Server/Web/System/BuildVersionHandler.cs [hunk 3]
```diff
@@ -49,5 +54,51 @@ private static byte[] GetVersionBuffer()
...
+                    context.Write(writer, new DynamicJsonValue
+                    {
+                        [nameof(LatestVersionCheck.VersionInfo.Version)] = versionUpdatesInfo?.Version,
+                        [nameof(LatestVersionCheck.VersionInfo.PublishedAt)] = versionUpdatesInfo?.PublishedAt,
+                        [nameof(LatestVersionCheck.VersionInfo.BuildNumber)] = versionUpdatesInfo?.BuildNumber
+                    });
+                }
+
+                var versionBuffer = stream.ToArray();
+                await ResponseBodyStream().WriteAsync(versionBuffer);
```
Comment (ppekrol) [inline src/Raven.Server/Web/System/BuildVersionHandler.cs]: not needed

**Review by ppekrol — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Web/System/BuildVersionHandler.cs (hunk 2 above)
Comment (gregolsky) [reply src/Raven.Server/Web/System/BuildVersionHandler.cs]: I could have sworn I changed those.. Updated both PRs.

**Pull request merged**